packages/roosterjs-editor-plugins/lib/plugins/TableResize/editors/TableEditor.ts

```diff
@@ -69,7 +76,8 @@ export default class TableEditor {
         private onShowHelperElement?: (
             elementData: CreateElementData,
             helperType: 'CellResizer' | 'TableInserter' | 'TableResizer' | 'TableSelector'
-        ) => void
+        ) => void,
+        private eventTarget?: EventTarget
```

Comment:
I only see eventTarget is used at line 98, so no need to add "private" to make it as a member